test-assets/detect-changes-configs/ntp-release/jobs/ntpd/spec~

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
Thanks.  I had been watching for them, and thought I had them all. Didn't remember to search in the 75+ file commit.